integration_tests/e2e/challenges/edit/editChallenge.cy.ts

```diff
@@ -0,0 +1,56 @@
+import Page from '../../../pages/page'
+import Error404Page from '../../../pages/error404'
+import ChallengesPage from '../../../pages/profile/challengesPage'
+import AuthorisationErrorPage from '../../../pages/authorisationError'
+
+context('Edit a Challenge', () => {
```

Comment:
Basic cypress tests (edit a challenge; and strength, condition, support strategy in the 3 files that follow) that are just enough to prove the routes and RBAC config for these edit stories.
Will flesh out these tests as we implement more code for each of them ....